File: LivrusVirtus/wxWindows/wx.NET/CS/Livrus.cs
using System;
using System.Drawing;
using wx;

namespace Livrus
{		
	public class LivrusApp : wx.App
	{
		public override bool OnInit()
		{
      RotinasGlobais.Rotinas = new RotinasGlobais();
      ConsultasSQL.ConsSQL = new ConsultasSQL();
			frmPrincipal.fPrincipal = new frmPrincipal();

      if (RotinasGlobais.Rotinas.ConectaBanco(
        "File Name=C:\\LivrusVirtus\\bancos\\access\\Access2k_Livrus.udl")==false)
        return false;

      frmSplash.fSplash = new frmSplash(null);
      frmSplash.fSplash.Show(true);

			return true;
		}
		//---------------------------------------------------------------------
		[STAThread]
		static void Main()
		{
			LivrusApp aplicativo = new LivrusApp();
			aplicativo.Run();
		}
	}	
}

File: LivrusVirtus/wxWindows/wx.NET/VB.NET/Livrus.vb
Option Explicit On 
Option Strict On

Imports System
Imports System.Drawing
Imports wx

Namespace Livrus

	Public Class LivrusApp
		Inherits wx.App

		Public Overrides Function OnInit() As Boolean
      RotinasGlobais.Rotinas = new RotinasGlobais()
      ConsultasSQL.ConsSQL = new ConsultasSQL()
      frmPrincipal.fPrincipal = New frmPrincipal()

      if not RotinasGlobais.Rotinas.ConectaBanco( _
        "File Name=C:\LivrusVirtus\bancos\access\Access2k_Livrus.udl") then
        return false
      end if
      frmSplash.fSplash = new frmSplash(nothing)
      frmSplash.fSplash.Show(true)

			Return True
		End Function

    <STAThread()> Shared Sub Main()
    		dim aplicativo as LivrusApp 
        aplicativo = New LivrusApp()
    		aplicativo.Run()
    End Sub
	End Class

End NameSpace
